thoth/metrics_exporter/jobs/db.py

```diff
@@ -78,6 +93,49 @@ def get_tot_relation_records_count(cls) -> None:
 
         _LOGGER.debug("thoth_graphdb_total_relation_records=%r", relation_models_records)
 
+    @classmethod
+    @register_metric_job
+    def get_bloat_data(cls) -> None:
+        """Get bloat data from database."""
+        if not cls._SCRAPE_COUNT == 0:
+            metric_name = "thoth_graphdb_last_evaluation_bloat_data"
+            metric = cls._PROM.get_current_metric_value(
+                metric_name=metric_name, label_config={"instance": cls._METRICS_EXPORTER_INSTANCE}
+            )
+
+            last_prometheus_scrape = datetime.fromtimestamp(int(metric[0]["value"][0]))
+            last_evaluation = datetime.fromtimestamp(int(metric[0]["value"][1]))
+
+            if (last_prometheus_scrape - last_evaluation).total_seconds() > timedelta(
+                days=cls._BLOAT_DATA_SCRAPE_INTERVAL_DAYS
+            ).total_seconds():
+                pass
+            else:
+                return
+
+        bloat_data = cls.graph().get_bloat_data()
+        metrics.graphdb_last_evaluation_bloat_data.set(datetime.utcnow().timestamp())
```

Comment:
I think it will be better to move this after setting the corresponding labels. If any fails, we will keep old last evaluation timestamp so this gathering retries.